app/ide-desktop/lib/dashboard/src/layouts/Drive.tsx

```diff
@@ -68,16 +72,29 @@ export interface DriveProps {
   readonly dispatchAssetListEvent: (directoryEvent: assetListEvent.AssetListEvent) => void
   readonly assetEvents: assetEvent.AssetEvent[]
   readonly dispatchAssetEvent: (directoryEvent: assetEvent.AssetEvent) => void
-  readonly setProjectStartupInfo: (projectStartupInfo: backendModule.ProjectStartupInfo) => void
-  readonly doOpenEditor: () => void
-  readonly doCloseEditor: (projectId: backendModule.ProjectId) => void
+  readonly doOpenEditor: (id: dashboard.Project['id']) => void
```

Comment:
prefer exporting `ProjectId` and using that instead just for consistency with `backend.ProjectId`